legal/beitragsordnung.md

```diff
@@ -0,0 +1,20 @@
+# Beitragsordnung
+
+Der Jahresbeitrag wird von den Mitgliedern selbst gewählt. Er beträgt jedoch mindestens
+1. 30 € für natürliche Personen,
+2. 500 € für Institute und sonstige juristische Personen. 
+
+Jedes Mitglied hat den Jahresbeitrag spätestens bis zum 31. Januar des Jahres
```

Comment:
Kommentar CK: ggf. besser. SEPA-Einzugsermäcjtigung, sonst läuft man den Beiträgen immer hinterher.
Alternativ: die Mitgelider verpflichten, einen daueraufrag einzurichten. Das hat dann den Vorteil, dass bei einem Kontowechsel, den sie sicherlich vergessen euch mitzutzeilen, sie selbst daran denken, den Dauerauftrag umzustellen.

Reply:
Bekommt man die Verpflichtung zum Dauerauftrag sauber hin? (Wäre tatsächlich in der Handhabung eigentlich auch für SEPA-Mandate. Allerdings gibt es dann ggf. kostenpflichtige Rücklastschriften.)